source/gui/settingsDialogs.py

```diff
@@ -2743,6 +2808,23 @@ def _onSoundVolChange(self, event: wx.Event) -> None:
 			wasapi
 			and not self.soundVolFollowCheckBox.IsChecked()
 		)
+		self.soundSplitComboBox.Enable(wasapi)
+		self.soundSplitModesList.Enable(wasapi)
+
+	def isValid(self) -> bool:
+		enabledSoundSplitModes = self.soundSplitModesList.CheckedItems
+		if len(enabledSoundSplitModes) < 1:
+			log.debugWarning("No sound split modes enabled.")
+			gui.messageBox(
+				# Translators: Message shown when no sound split modes are enabled.
+				_("At least one sound split mode has to be checked."),

```

Comment:
At least two modes for a cycling command:
```suggestion
				# Translators: Message shown when less than two sound split modes are enabled.
				_("At least two sound split mode have to be checked."),
```

Reply:
Again I see no point in forcing user to select the second mode if they don't want it. It is still going to cycle through 1 mode, nothing is going to break. Let's not add artificial restrictions on the user.

Reply:
My opinion is  that in this case cycling by a single option is justified. I don't think comparing this with the speech modes cycling makes sense, since the only way to change the mode is by a keyboard command, or restart of NVDA. For sound split you can have a default preferred mode and a second one which you need rarely, but switching to it has to be done fast. In that case you use the cycle command as a single way toggle and enable whatever mode you use as your main one from the GUI afterwards.

Reply:
So let's take an example to check we understand well each other:
* default preferred mode = disabled
* second mode rarely needed, but switching to it has to be done fast = NVDA left and other apps right.

The user uses the second mode when receiving a call, so that switching can be done very fast.

If there are two modes available in the cycling command, I do not know why switching would be less fast:
* if the user is already in split mode when the call arrives, there is nothing to do, so it's as fast as possible
* if the user is in default mode when the call arrives, just pressing NVDA+alt+S is needed, which is as fast as with only one mode in the cycling command.

Have I missed something?
I maintain that a cycling command with only one possible value is a confusing user experience. So if there is no use case where a one-value cycling command is more efficient than a two-value cycling command, there is no point in allowing it.

Reply:
@CyrilleB79 - I mentioned here: https://github.com/nvaccess/nvda/pull/16071#discussion_r1498438921

I agree, but would like to hear more feedback on this. The warning certainly doesn't hurt much, but I don't imagine anyone wanting remove "Off" from the list.